swapserverrpc/staticaddr.proto

```diff
@@ -195,44 +195,23 @@ message FetchSweeplessSweepTxResponse {
     // to.
     string sweep_addr = 1;
 
-    // The info the server used to generate the standard fee partial sweepless
-    // tx sigs.
-    ServerSweeplessSigningInfo standard_fee_info = 2;
-
-    // The info the server used to generate the high fee partial sweepless tx
-    // sigs.
-    ServerSweeplessSigningInfo high_fee_info = 3;
-
-    // The info the server used to generate the extreme fee partial sweepless tx
-    // sigs.
-    ServerSweeplessSigningInfo extreme_fee_info = 4;
+    // A map of fee rate to the nonces.
+    map<uint64, ServerSweeplessSigningInfo> fee_rate_to_nonces = 2;
```

Comment:
nit: i'd call this `server_signing_info` (and the one in the `PushStaticAddressSweeplessSigsRequest` `client_signing_info`.